api/api/src/main/scala/uk/ac/wellcome/platform/api/models/WorkFilter.scala

```diff
@@ -23,18 +16,7 @@ case class DateRangeFilter(fromDate: Option[LocalDate],
 case object IdentifiedWorkFilter extends WorkFilter
 
 case class LanguageFilter(languageIds: Seq[String]) extends WorkFilter
-object LanguageFilter extends ApplyCommaSeperated[LanguageFilter] {
-  val fromSeq = LanguageFilter(_)
-}
 
 case class GenreFilter(genreQuery: String) extends WorkFilter
 
 case class SubjectFilter(subjectQuery: String) extends WorkFilter
-
-trait ApplyCommaSeperated[T] {
-
-  protected val fromSeq: Seq[String] => T
-
-  def apply(str: String): T =
-    fromSeq(str.split(',').map(_.trim))
-}
```

Comment:
:point_up: not really needed to do all this since akka-http refactor, query parameters are parsed using circe